projects/igniteui-angular/src/lib/carousel/carousel.component.ts

```diff
@@ -64,78 +79,112 @@ export class IgxCarouselComponent implements OnDestroy {
      * Sets whether the carousel can pause the slide transitions.
      *
      * Default value is true
-     *
+     * ```html
+     *  <igx-carousel [pause]="false"></igx-carousel>
+     * ```
      * @memberOf IgxCarouselComponent
      */
     @Input() public pause = true;
-
+    /**
+     *Returns the time interval in milliseconds before the slide changes.
+     *```typescript
+     * let timeInterval = this.carousel.interval;
+     * ```
+     * @memberof IgxCarouselComponent
+     */
     @Input()
     get interval(): number {
         return this._interval;
     }
-
+    /**
+     *Sets the time interval in milliseconds before the slide changes.
+     *If not set, the carousel will not change slides automatically.
+     * ```html
+     * <igx-carousel interval = 1000></carousel>
+     * ```
+     * @memberof IgxCarouselComponent
+     */
     set interval(value: number) {
         this._interval = +value;
         this._restartInterval();
     }
-
+    /**
+     * Returns the value which indicates that the current slide is active.
+     * ```typescript
+     * this.slide
+     * ```
+     * @memberof IgxCarouselComponent
+     */
     @HostBinding('attr.tabindex')
     get tabIndex() {
         return 0;
     }
 
     /**
      * Controls whether the carousel should render the left/right navigation buttons.
-     *
      * Default value is true
```

Comment:
Please wrap true, false, other values cited in the comments, or names of inputs/outputs/methods/properties in ``:
Default value is `true`. This will provide highlighting in the output document.